samwell/sam/sambuilder.py

```diff
@@ -20,11 +20,13 @@
 from typing import List
 from typing import Optional
 from typing import Tuple
+from typing import IO
```

Comment:
nit: keeping the imports in alphabetical order
```suggestion
from typing import IO
from typing import List
from typing import Optional
from typing import Tuple
```